core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4092,9 +4107,12 @@ private void handleReady(long fileId) throws AlluxioException, IOException {
       // Lookup relevant file information.
       AlluxioURI uri;
       String tempUfsPath;
+      MountTable.Resolution resolution;
```

Comment:
Declare variable `resolution` in advance, in order to saving the resolve result in `try` block.